chicken-sloths/bangazon-site-sprint3
# Styling! Buttons working!

# Description
This PR touches a lot of files, so here's a list of what's happening:

`client/front-end.js`
1. Adds event listener for deleting a product from authed user's cart
1. Renames listeners to be looking for IDs

`controllers/authCtrl.js`
1. Redirects to `/` on login or register, not `/welcome`
1. Removes `console.log` stuff

`controllers/cartCtrl.js`
1. Provides a message if authed user has no items in cart
1. Provides property used to print delete buttons
1. Includes prices on those items in the cart

`views/cart.pug`
1. Prints message if there are no items in the cart

`views/index.pug`
1. Prints welcome message on login or register`

`views/...`
1. Generally restyling

`views/welcome.pug`
1. DELETED!

## Steps to Test Solution

1. Everything should continue working, EXCEPT the delete button on cart view (Kenzie's on that).
